tests/crds/k8s.cni.cncf.io_networkattachmentdefinitions_crd.yaml

```diff
@@ -1,57 +0,0 @@
-# Copyright 2020 NVIDIA
```

Comment:
This one is need for the unit-tests:
https://github.com/Mellanox/network-operator/blob/master/controllers/suite_test.go#L85
https://github.com/Mellanox/network-operator/blob/master/controllers/suite_test.go#L75